sdk/react/panels/CastDevicesScreen.js

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Animated,
+  ListView,
+  Text,
+  View,
+  TouchableHighlight
+} from 'react-native';
+
+import {
+  BUTTON_NAMES
+} from '../constants';
+import Utils from '../utils';
+
+import castScreenStyles from './style/castScreenStyles.json';
+const styles = Utils.getStyles(castScreenStyles);
+
+let ds = new ListView.DataSource({rowHasChanged: (r1, r2) => r1 === r2});
+
+const dismissButtonSize = 20;
+const castButtonSize = 35;
+
+class CastDevicesScreen extends Component {
+  static propTypes = {
+    height: PropTypes.number,
+    width: PropTypes.number,
+    onDismiss: PropTypes.func,
+    onDeviceSelected: PropTypes.func,
+    config: PropTypes.object,
+    deviceIds: PropTypes.array,
+    deviceNames: PropTypes.array,
+    selectedItem: PropTypes.string
+  };
+
+  componentWillMount() {
+    this.state = {
```

Comment:
`state` should be declared as a class field

Reply:
This particular class should not be checked since @atopilskii is working on it.
His branch https://github.com/ooyala/native-skin/tree/PLAYER-5035 will contain an updated version of `CastDeviceScreen.js`

Reply:
will it be merged in this branch?